pkg/toggles/toggle_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018 the Service Broker Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package toggles
+
+import (
+	"fmt"
+
+	"github.com/spf13/viper"
+)
+
```

Comment:
The examples are helpful, however there doesn't seem to be any unit tests verifying the logic within the toggles package.

Reply:
`go test` will validate the examples against the Output blocks at the end, I'm also happy to add other tests if you think they're warranted. I started with the examples because the tests seemed to fit that style better than the more declarative table driven one.

Reply:
I forgot about that feature, which I'm sure you've mentioned before. SGTM, again ;)